src/race.go

```diff
@@ -243,7 +243,7 @@ func (race *Race) Start3() {
 	if race.Ruleset.Format == "custom" {
 		time.Sleep(4 * time.Hour) // We need to make the timeout longer to accomodate multi-character speedrun races
 	} else {
-		time.Sleep(30 * time.Minute)
+		time.Sleep(90 * time.Minute) // An hour and a half to accomodate players that are not as good at the game
```

Comment:
Letting races last longer can be undesirable. Streamers like Molfried use the 30 minute limit as a reasonable cutoff so that he can accommodate both good players and bad. Also, it is common for racers to DC, or leave races open that never get finished, so longer timeouts will lead to a more cluttered lobby. As a compromise to racers who do want a longer timeout, the option is always available to use the custom format and just do a "normal" unseeded race.

Reply:
Commit has been removed.